docs: add ROADMAP.md (#8)
registry-operator/registry-operator: ROADMAP.md

# Roadmap

Welcome to the official roadmap for `registry-operator`! This document provides a high-level overview of our development plan and upcoming milestones. Whether you're a user, contributor, or simply curious about the future of the project, this roadmap aims to keep you informed. 

## Release 0.1.0

> In this phase of the project the focus is on the `registry-operator` scaffolding and configuring `Registry` with default configuration. The expected features are:
> 
> - 

Reviewer: ```suggestion
The focus is on the `registry-operator` scaffolding and deployment of the registry with default configuration.
```

## Release 0.2.0

> In this phase of the project the focus is on `s3` protocol configuration. The expected features are:
> 
> - 

Reviewer: ```suggestion
The focus is on bringing the `s3` protocol configuration to the registry via `Registry` resource.
```
